server/helpers/loadConfig.js

```diff
@@ -0,0 +1,85 @@
+// config intended to be passed on to frontend
+// keys in process .env must be referencesd explicitly
+// so dotenv-webpack is able to include them in the test bundle for Mocha tests
+const check = require('check-types')
+
+// process.env.npm_package_version is undefined on Heroku
+const appVersion = require('../../package.json').version
+
+const boolStrings = ['true', 'false']
+const isBoolString = (str) => boolStrings.includes(str)
+
+const {
+  ADALITE_ENABLE_DEBUGGING,
+  ADALITE_SERVER_URL,
+  ADALITE_BLOCKCHAIN_EXPLORER_URL,
+  ADALITE_DEFAULT_ADDRESS_COUNT,
+  ADALITE_GAP_LIMIT,
+  ADALITE_DEMO_WALLET_MNEMONIC,
+  ADALITE_ENABLE_TREZOR,
+  ADALITE_LOGOUT_AFTER,
+  ADALITE_ENABLE_SERVER_MOCKING_MODE,
+  ADALITE_MOCK_TX_SUBMISSION_SUCCESS,
+  ADALITE_MOCK_TX_SUMMARY_SUCCESS,
+  TREZOR_CONNECT_URL,
+  REDIS_URL,
+  STATS_PWD,
+  PORT,
+} = process.env
+
+const checkMap = check.map(process.env, {
+  ADALITE_ENABLE_DEBUGGING: isBoolString,
+  ADALITE_SERVER_URL: check.nonEmptyString,
+  ADALITE_BLOCKCHAIN_EXPLORER_URL: check.nonEmptyString,
```

Comment:
I would check whether it is an url, there is a validator for it in check-types AFAIK

Reply:
webUrl predicate was removed in check-types version 3